### PR TITLE
ADR for using a monorepo for SCM, but npm for datasets

### PR DESCRIPTION
**Pull request by mrchrisadams**
@mamhoff I've tried to outline the plan was described it here

### CODE REVIEW

File: doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Moroever, they need to be run in the AMEE platform. We want to be able require models individually, and re-use the package.json package management infrastructure for tracking dependencies, between datasets, so you only need to `npm install` the model you care about, then make the code as much like working with javascript, and convenient js datastructures as possible, liks JSON.
+
+## Decision
+
+The plan is to take to:
+
+- convert `item_definitions.csv` to an actual csv file
+- use this and `data.csv` to generate a `data.json` file
+- make a separate `package.json` for individual datasets
+- publish the datasets to npm so they can be installed separately, with an predictable API.
```
Comment (mamhoff) [inline doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md:37]: I think it's a good idea to actually keep the `.csv` files and generate the `data.json` when building the NPM modules. CSV is way easier to edit (just load it in a spreadsheet and save). JSON, especially when formatted such that it's easily sent over the wire, can be nasty to look at and requires programming skills and tools to edit.

File: doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Consequences
+
+We no longer need CSV parsing files. Code relying on the current `item_definitions.csv` format will not be able to consume new datasets.
```
Comment (mamhoff) [inline doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md]: Hm, well, I think we just restrict parsing them to building the NPM packages, but keep them as they are in the repo.

**Review by mamhoff — APPROVED** [review]
This is fantastic!

File: doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md (hunk 1 above)
Comment (mrchrisadams) [reply doc/adr/0001-have-a-single-repo-but-separate-installable-datasets.md:37]: Good point. I'll update the ADR to make this explicit.

**Pull request merged**